_posts/2019-01-23-dask-1.1.0.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: Dask Release 1.1.0
+tags: [release]
+theme: twitter
+---
+{% include JB/setup %}
+
+I'm pleased to announce the release of Dask version 1.1.0.  This is a major
+release with bug fixes and new features.  The last release was 1.0.0 on
+2018-11-29.
+This blogpost outlines notable changes since the last release.
+
+You can conda install Dask:
+
+    conda install dask
+
+or pip install from PyPI:
+
+    pip install dask[complete] --upgrade
+
+Full changelogs are available here:
+
+-  [dask/dask](https://github.com/dask/dask/blob/master/docs/source/changelog.rst)
+-  [dask/distributed](https://github.com/dask/distributed/blob/master/docs/source/changelog.rst)
+
+## Notable Changes
+
+A lot of work has happened over the last couple months, and we encourage people
+to look through the changelog to get a sense of the kinds of incremental
+changes that developers are working on.
+
+There are also a few notable changes in this release that we'll highlight here:
+
+-  Support for the recent Numpy 1.16 and Pandas 0.24 releases
+-  Support for Pandas Extension Arrays (see [Tom Augspurger's post on the topic](TODO))
+-  High level graph in Dask dataframe and operator fusion in simple cases
+-  Increased support for other libraries that look enough like Numpy and Pandas
```

Comment:
We you planning to add a section for this? If not, think mentioning / linking to cupy, sparse, and pygdf would be good.

Reply:
Handled.